containers-tests/tests/intset-properties.hs

```diff
@@ -457,6 +459,12 @@ prop_filter s i =
      valid evens .&&.
      parts === (odds, evens)
 
+prop_mapMaybe :: IntSet -> Property
+prop_mapMaybe s =
+  let f n = if odd n then Just n else Nothing
```

Comment:
Consider using an arbitrary `Fun` instead, makes the test simpler.
Same for the other `mapMaybe` property tests.